templates/deployment.yaml

```diff
@@ -43,6 +43,9 @@ spec:
               value: {{ quote .Values.config.requireAuthentication }}
             - name: AEROSPIKE_RESTCLIENT_POOL_SIZE
               value: {{ quote .Values.config.poolSize }}
+          {{- if .Values.config.extraEnvs }}
+            {{- toYaml .Values.config.extraEnvs | trim | nindent 12 }}
+          {{- end }}
```

Comment:
@reugn 
I think it would be good to add each config in REST client as a new config in the helm chart (even if it goes in as a ENV variable to the container). We can have each of these new config documented. We already did that for the other configurations. Let me know what you think.